reuse/contribute-to-docs.txt

```diff
@@ -28,18 +28,101 @@ Use the :guilabel:`Give feedback` button at the top of any page to create a GitH
+
+The documentation structure should look like this:
+
+.. code-block:: none
+
+   documentation/
+   ├── tutorial/
+   │   ├── subcategory-one/
+   │   │   ├── index.rst
+   │   │   ├── page-one.rst
+   │   │   ├── page-two.rst
+   │   │   └── page-three.rst
+   │   └── subcategory-two/
+   │   |   ├── index.rst
+   │   |   ├── page-one.rst
+   │   |   ├── page-two.rst
+   │   |   └── page-three.rst
+   |   └── index.rst
+   ├── how-to-guides
+   ├── explanation
+   ├── reference
+   └── index.rst
+
+Create new categories
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can create new categories to the documentation by following these steps:
+
+1. Create a new folder in your documentation directory.
+2. Create a new ``index.rst`` file within your new folder.
+3. Add the title of your new category to the first line of the ``index.rst`` file. Underline it using equal signs (=) that match the length of your title. For more information on titles and headings, read the `reStructuredText`_ style guide.
+4. In the ``index.rst`` file, add content introducing the category, its purpose, and other relevant links.
+5. In your ``index.rst`` file, add a toctree that specifies the file names of pages and the index files of the subcategories within your newly created category. The toctree should resemble the following structure:
+
+.. code:: rst
+
+   .. toctree::
+      :maxdepth: 2
+
+      subcategory-one/index
+      Subcategory two <subcategory-two/index>
+      page-one-file-name
+
+For more information, read the `Sphinx documentation on toctree`_.
+
+5. Update the documentation base ``index.rst`` file by adding your new category to the toctree.
+
+Create new subcategories
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To add a subcategory within an existing category in your documentation, follow these steps:
+
+1. Go to the parent category and create a new folder for your subcategory within it.
+2. Create an ``index.rst`` file within the subcategory folder.
+3. Enter the title of your new subcategory on the first line of the ``index.rst`` file. Underline it using equal signs (=) that match the length of your title. For more information on titles and headings, read the `reStructuredText`_ style guide.
+4. In the ``index.rst`` file, add content introducing the subcategory, its purpose, and other relevant links.
+5. In your ``index.rst`` file, add a toctree that includes the file names or titles of pages within your new subcategory. The toctree should resemble the following structure:
+
+.. code:: rst
+
+   .. toctree::
+      :maxdepth: 1
+
+      page-one-file-name
+      Page Two Title <page-two-file-name>
+
+.. note::
```

Comment:
This note is probably not needed since it is quite obvious that the names are examples used here and we have already pointed them to the Sphinx documentation on toctrees earlier.